vba_timeseries/models/nonu/suuvid_obs_nonu.m

```diff
@@ -13,8 +13,8 @@
 kappa = phi(3); %softmax temperature
 omega = phi(4); %stickiness
 
-tdiff = u(4); %cross-check position in u
-active_action = u(5); %cross-check position in u
+tdiff = u(3); %cross-check position in u
```

Comment:
This should stay 4 and 5 as indices of u that denote these input variables. Only phi indexing has changed, not u.